src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Integration/snapshots/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=controllers.verified.txt

```diff
@@ -66,11 +66,9 @@
                 "type": "object",
                 "properties": {
                   "Title": {
-                    "minLength": 5,
                     "type": "string"
```

Comment:
Was `minLength: 5` wrong so it is now omitted, or did the test change to remove minLength from this property?